generated/nifake/nifake_service.cpp

```diff
@@ -43,6 +43,13 @@ namespace nifake_grpc {
       output->AddAllocated(message);
     }
   }
+   CustomStruct NiFakeService::GetStructFromGrpcType(const nifake_grpc::FakeCustomStruct& input) 
```

Comment:
Looks like this got generated for `GetArrayForCustomCodeCustomType`, but function itself didn't get generated because of `codegen_method: no`. I guess that's fine because we shouldn't have any functions with `codegen_method: no` in our drivers.

Reply:
Yeah I think this API has input parameter of custom type, so this got generated. But this API is not being used anywhere, not sure why this is present in nifake.